mycore-restapi/src/main/java/org/mycore/restapi/MCRTransactionFilter.java

```diff
@@ -32,9 +32,9 @@
 @Priority(Priorities.USER)
 public class MCRTransactionFilter implements ContainerRequestFilter {
 
-    public static final String PROP_REQUIRE_TRANSACTION = "mcr:jpaTrans";
+    private static final Logger LOGGER = LogManager.getLogger(MCRTransactionFilter.class);
```

Comment:
please remove parameter and keep LogManager.getLogger(). The parameter is redundant and this line maybe copied without changing the parameter. Yes it happened before!